### ISSUE TITLE
How to do training with 'mmt create' without internal splitting on train/dev/test ?

### ISSUE THREAD

**Issue by cattoni** [modernmt/modernmt#496]
Dear MMTers

In previous versions it was possible to invocate 'mmt create' with the options
   --no-split --validation-corpora PATH_TO_DEV_SET
to avoid internal splitting of train data into train/dev/test (useful options when the user has their own dev and test data separated from the train data)

Now in v4.0 if I use such options 'mmt create' fails at step
(4/4) Training neural model
    (1/2) Train neural network...   
and modernmt/runtime/ENGINE/logs/training.log reports

> fairseq-train: error: unrecognized arguments: --no-split --validation-corpora ../data/default_my_split/dev

Am I using the correct options?

**Comment by nicolabertoldi** [comment]
@cattoni 
with v4 it is possible to skip the extraction of a test set using `--no-test` (see help)

Instead it is not possible to skip the extraction of the dev set; the old parameter `--validation-corpora ` was dismissed.

If you think this is an important feature, please mark this ticket with the label "feature-request"
and give a motivation.

**Comment by cattoni** [comment]
Thanks @nicolabertoldi 
I update my scripts with the new option.
